app/presenters/updatable.rb

```diff
@@ -20,13 +25,21 @@ def history
     end
   end
 
-private
+  # The direction of change history isn’t guaranteed
+  # https://github.com/alphagov/govuk-content-schemas/issues/545
+  def reverse_chronological_change_history
+    change_history.sort_by { |item| DateTime.parse(item[:timestamp]) }.reverse
+  end
 
   def any_updates?
-    if (first_public_at = content_item["details"]["first_public_at"]).present?
+    if first_public_at.present?
       DateTime.parse(content_item["public_updated_at"]) != DateTime.parse(first_public_at)
     else
       false
     end
   end
+
+  def first_public_at
+    content_item["details"]["first_public_at"]
+  end
```

Comment:
All other formats that display published/updated uses `first_public_at` at the moment. I think it's ok to say "specialist is different" and leave as-is for now. Might need to re-consider when we bring in other formats.

When the data upstream is fixed we'll only need to change it in two places.